packages/jbrowse-web/src/Loader.tsx

```diff
@@ -14,6 +14,7 @@ import { PluginConstructor } from '@gmod/jbrowse-core/Plugin'
 import corePlugins from './corePlugins'
 import JBrowse from './JBrowse'
 import JBrowseRootModelFactory from './rootModel'
+import packagedef from '../package.json'
```

Comment:
Will this work when it's built? I don't think package.json ends up in the build.

Reply:
it gets bundled via a static file loader not dynamic import and looks like it works in the build